rules/private/intellij_compiler_xml.bzl

```diff
@@ -37,6 +37,27 @@ def _impl(ctx):
         ],
         progress_message="Generating intellij compiler.xml file: %s" % ctx.outputs.compiler_xml_file.path)
 
+    shell_script_content = \
+"""#!/bin/bash -e
+
+if [ "$1" = "--force" ]; then
+    rm -f compiler.xml
+else
+    if [ -f compiler.xml ]; then
+        echo "Refusing to overwrite $(pwd)/compiler.xml" > /dev/stderr
```

Comment:
should this be ctx.outputs.compiler_xml_installer_srcipt_file ? or is pwd ok to use here?